q2_mlab/plugin_setup.py

```diff
@@ -1,152 +1,180 @@
+        "multinomial distribution instead of rarefying "
+        "without replacement.",
+        "n_jobs": sklearn_n_jobs_description,
     },
     output_descriptions={
-        'filtered_rarefied_table': 'The resulting filtered and rarefied '
-                                   'feature table.',
-        'filtered_metadata': 'The resulting filtered metadata containing '
-                             'only the specific target column.',
-        'jaccard_distance_matrix':
-            'Matrix of Jaccard distances between pairs of samples.',
-        'bray_curtis_distance_matrix':
-            'Matrix of Bray-Curtis distances between pairs of samples.',
-        'jensenshannon_distance_matrix':
-            'Matrix of Jensen-Shannon distances between pairs of samples.',
-        'aitchison_distance_matrix':
-            'Matrix of aitchison distances between pairs of samples.',
-        'unweighted_unifrac_distance_matrix':
-            'Matrix of unweighted UniFrac distances between pairs of samples.',
-        'weighted_unifrac_distance_matrix':
-            'Matrix of weighted UniFrac distances between pairs of samples.',
+        "filtered_rarefied_table": "The resulting filtered and rarefied "
+        "feature table.",
+        "filtered_metadata": "The resulting filtered metadata containing "
+        "only the specific target column.",
+        "jaccard_distance_matrix": "Matrix of Jaccard distances between pairs of samples.",
+        "bray_curtis_distance_matrix": "Matrix of Bray-Curtis distances between pairs of samples.",
+        "jensenshannon_distance_matrix": "Matrix of Jensen-Shannon distances between pairs of samples.",
+        "aitchison_distance_matrix": "Matrix of aitchison distances between pairs of samples.",
+        "unweighted_unifrac_distance_matrix": "Matrix of unweighted UniFrac distances between pairs of samples.",
+        "weighted_unifrac_distance_matrix": "Matrix of weighted UniFrac distances between pairs of samples.",
     },
-    name='Dataset preprocessing for benchmarking',
-    description=('Applies filtering and preprocessing steps '
-                 'to a feature table and metadata, and '
-                 'generates distance matrices with phylo- '
-                 'genetic and non-phylogenetic metrics.')
+    name="Dataset preprocessing for benchmarking",
+    description=(
+        "Applies filtering and preprocessing steps "
+        "to a feature table and metadata, and "
+        "generates distance matrices with phylo- "
+        "genetic and non-phylogenetic metrics."
+    ),
 )
 
-plugin.pipelines.register_function(
-    function=q2_mlab.benchmark_classify,
+algorithm_choices = [
+    "KNeighborsClassifier",
+    "RandomForestClassifier",
+    "GradientBoostingClassifier",
+    "XGBClassifier",
+    "RidgeClassifier",
+    "KNeighborsRegressor",
+    "RandomForestRegressor",
+    "GradientBoostingRegressor",
+    "XGBRegressor",
+    "RidgeRegressor",
+    "LinearSVR",
+    "LinearSVC"
+]
```

Comment:
For consistency's sake can you replace this for:

```suggestion
algorithm_choices = RegressionTask.algorithms | ClassificationTask.algorithms
```

Reply:
Thanks!